From: Jiri Slaby <jslaby@suse.cz>
Subject: kABI: protect struct vb2_queue
Patch-mainline: never, kabi

In 3.12.31, commit 1d5b4f75c30a69df306e53869d44206555147c62 (media:
vb2: fix VBI/poll regression), upstream commit
58d75f4b1ce26324b4d809b18f94819843a98731 added one member to struct
vb2_queue. It indeed changed the layout of that structure.

However the member is only a bitfiled member, and since there is
another one, it uses the hole in there.  So protect the change in the
structure the usual way, by #ifdef __GENKSYMS__.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 include/media/videobuf2-core.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/include/media/videobuf2-core.h
+++ b/include/media/videobuf2-core.h
@@ -356,7 +356,9 @@ struct vb2_queue {
 	unsigned int			plane_sizes[VIDEO_MAX_PLANES];
 
 	unsigned int			streaming:1;
+#ifndef __GENKSYMS__
 	unsigned int			waiting_for_buffers:1;
+#endif
 
 	struct vb2_fileio_data		*fileio;
 };
